Documentation/Usage.md

```diff
@@ -250,22 +254,16 @@ let jpegData = jpeg_common_struct()
 print(jpegData)
 ```
 
-Install JPEG library using a system packager, e.g, `$ brew install jpeg`
-
-Create a directory called `CJPEG` next to the `example` directory and
-initialize it as a package that builds a system module:
+Install JPEG library using a system packager, e.g, `$ brew install jpeg`.
+`jpeg` is a keg-only formula, meaning it won't be linked to `/opt/homebrew/lib`,
+and you'll have to link it manually at build time.
 
-    example$ cd ..
-    $ mkdir CJPEG
-    $ cd CJPEG
-    CJPEG$ swift package init --type empty
-
-This creates `Package.swift` file in the directory.
-Create a `module.modulemap` file so it consists of the following:
+Just like in the previous example, `mkdir Sources/CJPEG`, and add the
+following `module.modulemap`:
 
     module CJPEG [system] {
         header "shim.h"
-        header "/usr/local/include/jpeglib.h"
+        header "/opt/homebrew/opt/jpeg/include/jpeglib.h"
```

Comment:
ditto re. `/usr/local` with note that homebrew users this may be `/opt/homebrew/`